scrapers/Brazzers.yml

```diff
@@ -8,6 +8,7 @@ sceneByURL:
       - brazzers.com/video
       - men.com/sceneid/
       - realitykings.com/scene/
+      - babes.com/scene/
```

Comment:
realitykings and babes.com no longer work with cdp/this scraper
Most of the Mindgeeks sites probably have to move to the MindGeekAPI scraper anyway

Reply:
removed realitykings.com and babes.com